### ISSUE TITLE
New release to pypi (1.12.0)

### ISSUE THREAD

**Issue by mistercrunch** [dpgaspar/Flask-AppBuilder#808]
Last release was May 19, 2018, and we've had some major security issues fixed since then. I'd love for both Superset and Airflow to get bumped up to a more secure version of FAB.

**Comment by mistercrunch** [comment]
ping 🏓

**Comment by mistercrunch** [comment]
Had conversations with people in my communities (Airflow and Superset) and this is becoming a real issue for us. The scary word "fork" came up multiple times.

Let's fix governance here. @dpgaspar please be reasonably responsive on releasing to Pypi, or grant the power/responsibility to others to do so.

**Comment by dolfandringa** [comment]
@mistercrunch what security issues are you talking about specifically?

**Comment by mistercrunch** [comment]
@dolfandringa https://github.com/dpgaspar/Flask-AppBuilder/pull/797

**Comment by dolfandringa** [comment]
I'd like to help too if help is needed.

**Comment by dpgaspar** [comment]
Hi @mistercrunch,

Can you send me an Email to danielvazgaspar@gmail.com so we can discuss this issue?

**Comment by dpgaspar** [comment]
Yes, again so sorry everyone, I have the best intentions, but really no time to properly maintain an active project like this. Any interested contributer please email me.

One possible problem is with pypi, is there a proper way to share it?

**Comment by dolfandringa** [comment]
@dpgaspar I just emailed you. You can add collaborators in pypi to a package, which will enable them to upload too.

**Comment by dolfandringa** [comment]
Awesome! Thanks @dpgaspar
